com.unity.robotics.urdf-importer/Editor/CustomEditors/UrdfInertialEditor.cs

```diff
@@ -22,27 +23,64 @@ public class UrdfInertialEditor : UnityEditor.Editor
     {
         private Vector3 testVector;
 
+        bool ResetInertialsButton()
+        {
+            return GUILayout.Button("Reset Overrides");
+        }
+
         public override void OnInspectorGUI()
         {
             UrdfInertial urdfInertial = (UrdfInertial) target;
-
-            GUILayout.Space(5);
-            urdfInertial.displayInertiaGizmo =
-                EditorGUILayout.ToggleLeft("Display Inertia Gizmo", urdfInertial.displayInertiaGizmo);
+            
             GUILayout.Space(5);
-
-            bool newValue = EditorGUILayout.BeginToggleGroup("Use URDF Data", urdfInertial.useUrdfData);
-            EditorGUILayout.Vector3Field("URDF Center of Mass", urdfInertial.centerOfMass);
-            EditorGUILayout.Vector3Field("URDF Inertia Tensor", urdfInertial.inertiaTensor);
-            EditorGUILayout.Vector3Field("URDF Inertia Tensor Rotation",
-                urdfInertial.inertiaTensorRotation.eulerAngles);
+            var shouldOverride = 
+                EditorGUILayout.BeginToggleGroup("Override URDF Data", !urdfInertial.useUrdfData);
+            EditorGUI.BeginChangeCheck();
+            var centerOfMass = 
+                EditorGUILayout.Vector3Field("URDF Center of Mass", urdfInertial.centerOfMass);
+            var inertiaTensor = 
+                EditorGUILayout.Vector3Field("URDF Inertia Tensor", urdfInertial.inertiaTensor);
+            var inertialChanged = EditorGUI.EndChangeCheck();
+            EditorGUI.BeginChangeCheck();
+            var eulerAngles = 
+                EditorGUILayout.Vector3Field("URDF Inertia Tensor Rotation",
+                    urdfInertial.inertiaTensorRotation.eulerAngles);
+            var anglesChanged = EditorGUI.EndChangeCheck();
+            
+            var shouldReset = ResetInertialsButton();
             EditorGUILayout.EndToggleGroup();
 
-            if (newValue != urdfInertial.useUrdfData)
+            var toggleOccured = urdfInertial.useUrdfData == shouldOverride;
```

Comment:
if "UseURDFData" equals to "ShouldOverride"? 

Is the code set up in a way that whenever these two values are equal, that means there was a toggle?